components/raftstore/src/store/peer.rs

```diff
@@ -4989,6 +5054,37 @@ where
         }
     }
 
+    pub fn snapshot_recovery_maybe_finish_wait_apply(&mut self, force: bool) {
+        if let Some(SnapshotRecoveryState::WaitLogApplyToLast { target_index, .. }) =
+            &self.snapshot_recovery_state
+        {
+            // if (! self.is_leader()) && self.is_handling_snapshot() {
+            //     info!("snapshot recovery follower waiting apply snapshot";
+            //     "region_id" => self.region().get_id(),
+            //     "peer_id" => self.peer_id(),
+            //     "target_index" => target_index,
+            //     "applied" =>  self.raft_group.raft.raft_log.applied,
+            //     "force" => force,
+            //     );
+            //     return;
+            // }
```

Comment:
Please remove all unused code instead of commenting them.

Reply:
fixed